# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webrtc/libwebrtcglue/VideoConduit.h
# Commit: c5502287708e
# Full Hash: c5502287708e8ff072b205bb2308e04037b72bef
# Author: Dan Minor <dminor@mozilla.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 2
# Description:
#   Bug 1654112 - Enable more VideoConduit code. r=ng
#   
#   This enables more of the VideoConduit code and updates the APIs used.
#   
#   Note that this removes REMB support (upstream has removed the corresponding
# ==============================================================================

diff -r 99e7f86e5cad -r c5502287708e dom/media/webrtc/libwebrtcglue/VideoConduit.h
--- a/dom/media/webrtc/libwebrtcglue/VideoConduit.h	Thu Dec 03 07:34:49 2020 -0500
+++ b/dom/media/webrtc/libwebrtcglue/VideoConduit.h	Tue Dec 15 13:51:00 2020 -0500
@@ -581,7 +581,7 @@
   int mMinBitrate = 0;
   int mStartBitrate = 0;
   int mPrefMaxBitrate = 0;
-  // int mNegotiatedMaxBitrate = 0;
+  int mNegotiatedMaxBitrate = 0;
   int mMinBitrateEstimate = 0;
 
   // Set to true to force denoising on.
